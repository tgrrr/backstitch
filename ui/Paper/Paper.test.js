import React from 'react'
import renderer from 'react-test-renderer'
import { Paper } from '.'

it('Paper: default', () => {
    const component = renderer.create(
        <Paper>
            <>
                Paper
            </>
        </Paper>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
