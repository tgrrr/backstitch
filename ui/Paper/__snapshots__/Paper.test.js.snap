// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Paper "
    data-testid="Paper"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      Paper
    </div>
  </div>
</div>
`;
